src/components/container/HeartEssentialEight/HeartEssentialEight.css

```diff
@@ -0,0 +1,107 @@
+
+/* TODO, make the below a new component for users who might want to, e.g, put only the score overview on a dashboard, and then drill into another view for the list */
+
+.mdhui-heart-e8-score-container {
+  padding: var(--mdhui-padding-md);
+}
+.mdhui-heart-e8-score-container .le8-header {
+  margin: 0 0 var(--mdhui-padding-md) 0;
+}
+.mdhui-heart-e8-score-container .score {
+  font-size: 65px;
+  font-weight: bold;
+  display: flex;
+  justify-content: center;
+}
+
+.mdhui-heart-e8-score-container .scoreSubText {
+  font-size: 20px;
+  font-weight: bold;
+  color: rgb(109, 110, 113);
+  display: flex;
+  justify-content: center;
+}
+
+/* TODO: where is this used? */
+
+.mdhui-heart-e8-feedback {
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  margin-left: 20px;
+}
+
+/* TODO, make the below a new component for users who might want to, e.g, put only the score overview on a dashboard, and then drill into another view for the list */
+
+.mdhui-heart-e8-achievement-container {
+  padding: var(--mdhui-padding-xs) var(--mdhui-padding-md);
+}
+.mdhui-heart-e8-achievement-container .le8-header {
+  margin: var(--mdhui-padding-sm) 0 var(--mdhui-padding-xs) 0;
+}
+.achievementActionList {
```

Comment:
I think you will probably want to prefix all these classes with `mdhui-heart-e8-` to avoid any potential collisions.